src/main/java/com/zenjava/javafx/maven/plugin/GenerateKeyStoreMojo.java

```diff
@@ -248,4 +240,26 @@ protected void generateKeyStore(File keyStore,
         );
     }
 
+    private void checkKeystoreRequiredParameter(String value, String valueName) throws MojoExecutionException {
+        if (StringUtils.isEmpty(value)) {
+            throw new MojoExecutionException("The property '" + valueName + "' is required to generate a new KeyStore.");
+        }
+    }
+
+    private void checkAndAddRequiredField(StringBuilder distinguishedName, String spacer, String propertyName, String value, String fieldName, RequiredFieldAlternativeCallback alternative) throws MojoExecutionException {

```

Comment:
I think `propertyName` should contain only property name (without `=`). This method can add `=` inside.